test/ess-test-r.el

```diff
@@ -1,12 +1,31 @@
+;;; ess-test-r.el --- ESS tests for R  -*- lexical-binding: t; -*-
+;;
+;; This file is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation; either version 2, or (at your option)
+;; any later version.
+;;
+;; This file is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; A copy of the GNU General Public License is available at
+;; https://www.r-project.org/Licenses/
+;;
+;;; Commentary:
+;;
 
 (require 'ert)
 (require 'ess-r-mode)
-(require 'ess-r-tests-utils)
+(require 'ess-test-r-utils)
```

Comment:
General comment: I think this kind of stuff should be namespaced under `ess-r-test-`. I think it's a good principle to develop R stuff as if it were a separate package. Not a big deal though, we can rename all of this later again if/when we extract the R mode as its own package.

Reply:
Counter to what I said above, most of these tests are integration tests for ESS rather than unit tests for ESSR. I.e. we test the inferior code mostly through R.

Reply:
I imagine if/when we split out languages it'll involve a decent amount of renaming anyway.